_projects/it/datigov.md

```diff
@@ -45,6 +45,8 @@ In attesa di pubblicare altra documentazione relativa al progetto, qui per ora p
 
 [Documento d'architettura dell'evoluzione di dati.gov.it](https://docs.google.com/document/d/1dCfbpwmkl-U2kreykRY8YXJct6GOe-r_3qjism5wI2Y/edit)
 
+Inoltre, sono state pubblicate [Le linee guida tecniche per i cataloghi dati](http://linee-guida-cataloghi-dati-profilo-dcat-ap-it.readthedocs.io/it/latest/index.html) che forniscono (i) indicazioni puntuali sulla specifica del profilo di metadatazione DCAT-AP_IT, con note d'uso ed esempi pratici, (ii) i riferimenti all'ontologia OWL del profilo DCAT-AP_IT, (iii) mapping dei temi per i dati e dei sottotemi, nonché il mapping tra i temi per i dati e quelli INSPIRE relativi ai dati geospaziali e (iv) un primo elenco di vocabolari controllati usati nel profilo di metadatazione.
```

Comment:
Applicando questa patch, vedo così:

<img width="635" alt="screenshot 2017-04-07 17 11 39" src="https://cloud.githubusercontent.com/assets/1014109/24806477/4fa91362-1bb5-11e7-9f05-02753ba8ef61.png">

Direi che è meglio mantenere la struttura usata per gli altri link, cioè lasciare il link su una riga separata, e aggiungere un testo sopra che lascerei un filo più breve e discorsivo. Le informazioni più di dettagli potresti inserirle nella prima pagina del documento stesso, in modo che facciano da overview veloce al documento.

Come ultima nota, i link a RTD è meglio troncarli sul nome del progetto (quindi solo `http://linee-guida-cataloghi-dati-profilo-dcat-ap-it.readthedocs.io`) in modo che poi venga fatto il redirect al branch corretto in base alla lingua e alla configurazione (se il documento fosse in più lingue, sceglierebbe la lingua preferita dell'utente).